src/frontend/utils/date.ts

```diff
@@ -0,0 +1,7 @@
+import { parseISO } from 'date-fns';
+
+export const datoFormat = { day: '2-digit', month: '2-digit', year: 'numeric' };
+
+export const tilLokalDatoStreng = (datostreng: string): string => {
```

Comment:
Det ble lagt inn metoder for en uke siden som erstattet de som fantes i formatter
https://github.com/navikt/familie-ef-sak-frontend/blame/master/src/frontend/utils/formatter.ts 

Fint hvis dere merger inn master og eventuellt renamer de :)

Reply:
done